Source/Text/SvgTextBase.Drawing.cs

```diff
@@ -20,10 +20,11 @@ public override RectangleF Bounds
                 var path = this.Path(null);
                 foreach (var elem in this.Children.OfType<SvgVisualElement>())
                 {
-                    //When empty Text span, don't add path
+                    //If string is blank, GraphicsPath points will be empty.
```

Comment:
Please add space.
`// If string is blank, GraphicsPath points will be empty.`

Reply:
Ok. I'm done.